specs/agents/log-streaming.md

```diff
@@ -0,0 +1,48 @@
+# Log streaming
```

Comment:
Streaming is a technical term and relates to how data is sent. Like streaming vs batching. Some of agents don't stream events via chunked-encoding but send it in batches.

I think we should use a less technical term like log sending or log shipping that explains what the feature does instead of how it's implemented.